examples/snippets/utils/client/manual_auth.py

```diff
@@ -0,0 +1,22 @@
+"""
```

Comment:
It looks like this snippet from another PR snuck in.  Can you remove it so that it isn't a part of this branch?